action/store/postgres.py

```diff
@@ -5,10 +5,13 @@
 
 
 class PostgresStore(Store):
-    def __init__(self, name: str, database: str, user: str, password: str):
+    def __init__(
+        self, name: str, database: str, user: str, password: str, exclude_tables: list
+    ):
         self.meta = {
             "name": name,
             "type": "postgres",
+            "excluded_tables": exclude_tables,
```

Comment:
this will result in a new json entry in the data.json 
>
            "excluded_tables": [
                "transactions_migrations_goose"
            ]

if this is not desirable, i can remove it.
cc @gouline

Reply:
I can't think of any reason we'd want it there, so please exclude